docker/influxdb/downsampling.sql

```diff
@@ -60,8 +60,8 @@ BEGIN
         mean(percentile_90) as percentile_90,
         mean(percentile_95) as percentile_95,
         mean(stddev) as stddev
-    INTO "metrics"."1m".:MEASUREMENT
-    FROM metrics."10s"./.*/
+    INTO "metrics"."duration14d_precision_1m".:MEASUREMENT
```

Comment:
Space missing: duration_14d_precision_1m

Reply:
fixed, thank you